pagopa/pn-frontend
# Pn 2763 fe multilingua sul portale landing

## Short description
Update mui-italia to version 0.8.10.
Change pn-commons Footer to use language types from mui-italia LangSwitch component.
Add support for German (de), French (fr) and Slovenian (sl) languages on landing site.

## List of changes proposed in this pull request
- update mui-italia to v0.8.10
- refactor pn-commons Footer component to use types now exported by LangSwitch component
- add support to de, fr and sl LangCodes to landing site content manager
- add footer translation to landing site content manager
- add  en, de, fr and sl translations for both sections ("Cittadini" and "Enti") of the landing site

## How to test
Run the pn-landing application and verify you can choose any of the supported languages and see the page content changing accordingly.
